### PR TITLE
Remove --pre from notebooks after release.

### PR DESCRIPTION
**Pull request by MichaelBroughton**
For review after #5134

### CODE REVIEW

File: cirq-core/cirq/contrib/quimb/Contract-a-Grid-Circuit.ipynb [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "    import cirq\n",
     "except ImportError:\n",
     "    print(\"installing cirq...\")\n",
-    "    !pip install --quiet cirq --pre\n",
+    "    !pip install --quiet cirq\n",
```
Comment (maffoo) [inline cirq-core/cirq/contrib/quimb/Contract-a-Grid-Circuit.ipynb:15]: Might be nice to combine this with the following code which tries to import quimb and then installs more packages:
```python
try:
    import cirq
    import quimb
except ImportError:
    print("installing cirq-core[contrib]...")
    !pip install --quiet cirq-core[contrib]
    print("installed cirq-core[contrib].")
```

File: docs/tutorials/basics.ipynb [hunk 2]
```diff
@@ -98,7 +98,7 @@
     "    import cirq\n",
     "except ImportError:\n",
     "    print(\"installing cirq...\")\n",
-    "    !pip install --quiet cirq --pre\n",
+    "    !pip install --quiet cirq\n",
```
Comment (maffoo) [inline docs/tutorials/basics.ipynb:101]: This basic notebook almost certainly needs just `cirq-core`.

**Review by maffoo — COMMENTED** [review]
I have some questions about whether we can avoid installing all of cirq, otherwise this seems fine.

**Comment by MichaelBroughton** [comment]
I think I'm personally in favor of keeping the full install. Mostly for the sake of uniformity and a user not getting caught off guard if something is missing from a reduced install.

**Review by maffoo — APPROVED** [review]

**Pull request merged**